mmf/trainers/core/training_loop.py

```diff
@@ -164,10 +164,25 @@ def run_training_epoch(self) -> None:
 
     def run_training_batch(self, batch: Dict[str, Tensor], loss_divisor: int) -> None:
         report = self._forward(batch)
+        if self.training_config.exit_on_nan_losses:
```

Comment:
Probably add a simple test for this following existing trainer tests if possible?

Reply:
Sounds good!